boards/st/nucleo_h7s3l8/Kconfig.sysbuild

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2024-2025 STMicroelectronics
+# Copyright (c) 2025 JB-Lighting Lichtanlagentechnik
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NUCLEO_H7S3L8_STM32H7S3XX_EXT_FLASH_APP
+
+choice BOOTLOADER
+	default BOOTLOADER_MCUBOOT
+endchoice
+
+choice BOOT_SIGNATURE_TYPE
+	default BOOT_SIGNATURE_TYPE_NONE
+endchoice
+
+choice MCUBOOT_MODE
+	default MCUBOOT_MODE_SINGLE_APP
```

Comment:
direct xip?

Reply:
they are different modes, single slot means you use slot 0 and cannot do any updates from the app, MCUboot must have serial recovery mode to update slot 0, with direct-xip you can load firmware updates to the opposing slot and MCUboot will boot the highest version